pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -1866,3 +1897,19 @@ func getTpccLabels(
 
 	return labels
 }
+
+// This function converts exporter.SnapshotTick to openmetrics into a buffer
+func getHistogramOpenmetrics(
+	exporter exporter.Exporter, snapshots map[string][]exporter.SnapshotTick, perfBuf *bytes.Buffer,
```

Comment:
`perfBuf` is unused

Reply:
Oops, missed this!
Thanks for pointing out !